content/docs/0.16.x/integrations/webhooks/_index.md

```diff
@@ -0,0 +1,318 @@
+
+*An example of a customized request payload:*   
+
+```
+{
+  "text": "Evaluation in {{.data.stage}} finished with result {{.data.evaluation.result}} and score {{.data.evaluation.score}}."
+}
+```
+
+Based on the Go templating capabilities, you can: 
+
+* Define conditions: `"{{if .fieldName}}{{.fieldName}}{{ else }}No field name set{{ end }}"` 
+* Access an array element: `"{{ index .articles.Content 0 }}"`
+* Refer to attributes with non-string characters `"(index .data "incident-management-tool").url"`
+
+*An example of a customized request payload using a condition on an array element:*
+
+```
+{
+  "deploymentURL": "{{if index .data.deployment.deploymentURIsPublic 0}}{{index .data.deployment.deploymentURIsPublic 0}}{{else}}No deployment URL provided{{end}}"
+}
+```
+
+A convenient way to add event properties to the URL or custom payload is the property picker. 
+
+To use this, put your cursor in the URL or text field at the spot where you would like to customize the payload. Clicking the *computer* icon will open a list of data fields you can add to the payload or URL. This list of data fields is derived from the event your webhook is subscribed to. 
+
+{{< popup_image
+link="./assets/customize-payload.png"
+caption="Select event data to add to the payload or URL"
+width="700px">}}
+
+**Step 4: Set "Send Finished Event" flag**
+
+The `Send finished` event flag is not applicable to a *passive* webhook. So you can leave it unselected as shown below.
+
+{{< popup_image
+link="./assets/finished-event-empty.png"
+caption="Configure"
+width="400px">}}
+
+**Step 5: Create subscription**
+
+Click the **Create subscription** button to save and enable the webhook for your integration.
+
+## Configure Passive on task "triggered" events
+
+For this use case, the Keptn webhook service sends both the task `started` and task `finished` events to complete the task sequence. 
+
+An example *Passive* webhook is setting a feature flag where the Third-party webhook receiver can't send confirmation that it set the flag.
+
+**Add subscription**
+
+1. Follow the same steps above to add a new webhook subscription, but configure it to listen to a Task suffix of `triggered`. 
+1. Ensure that the `Automatically` option is selected as shown below. When this is set, the webhook service will automatically send the task `finished` event for your subscription.
+
+    {{< popup_image
+    link="./assets/finished-event-auto.png"
+    caption="Configure"
+    width="400px">}}
+
+1. Click the **Create subscription** button to save and enable the webhook for your integration.
+
+## Configure Active webhook on task "triggered" events
+
+This type is called *Active* because the Keptn webhook service will first send the task `started` event and the third-party webhook receiver sends back the `finished` task event to complete the task sequence.
```

Comment:
```suggestion
This type is called *Active* because the Keptn webhook service first sends the task `started` event and the third-party webhook receiver sends back the `finished` task event to complete the task sequence.
```